docs/source/status.rst

```diff
@@ -128,7 +128,7 @@ IPC Format
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
 | Sparse tensors              | ✓     |       |       |            |       |       |       |
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
-| Buffer compression          | ✓     | ✓ (3) | ✓     |            |       |       | ✓     |
+| Buffer compression          | ✓     | ✓ (3) | ✓     |            |       |  ✓    | ✓     |
```

Comment:
This was just added by @liukun4515 and @alamb in https://github.com/apache/arrow-rs/pull/2369

Reply:
🎉